### DISCUSSION TITLE
Running from a subdirectory?

### DISCUSSION THREAD

**Discussion by kerberjc** [sct/overseerr#895] [Help] [answered]
Any chance there is a configuration that can be made to run this out of a subdirectory?  I'd like to have it on the same subdomain as my radarr and sonarr run out of, e.g.:
media.domain.com/overseerr
media.domain.com/sonarr
media.domain.com/radarr

**Comment by sct** [comment, marked as answer]
There is an issue for this: https://github.com/sct/overseerr/issues/274

**Reply by kerberjc** [reply]
Perfect, thanks!
